src/Sylius/Bundle/CoreBundle/Form/Type/CartItemType.php

```diff
@@ -34,7 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
 
-        if (isset($options['product']) && $options['product']->hasVariants()) {
+        if (isset($options['product']) && $options['product']->hasVariants() && !$options['product']->isSimple()) {
```

Comment:
@bretto36 actually you added it back 😄  can you remove `isset` and squash your commits? (8 commits for 2 lines change looks strange :))